pkg/env/armhelper.go

```diff
@@ -110,24 +109,20 @@ func newARMHelper(ctx context.Context, log *logrus.Entry, env Interface) (ARMHel
 		armAuthorizer = autorest.NewBearerAuthorizer(sp)
 	}
 
-	fpGraphAuthorizer, err := env.FPAuthorizer(env.TenantID(), env.Environment().GraphEndpoint)
-	if err != nil {
-		return nil, err
-	}
-
 	return &armHelper{
 		log: log,
 		env: env,
 
 		roleassignments: authorization.NewRoleAssignmentsClient(env.Environment(), env.SubscriptionID(), armAuthorizer),
-		applications:    graphrbac.NewApplicationsClient(env.Environment(), env.TenantID(), fpGraphAuthorizer),
 	}, nil
 }
 
 func (ah *armHelper) EnsureARMResourceGroupRoleAssignment(ctx context.Context, fpAuthorizer refreshable.Authorizer, resourceGroup string) error {
 	ah.log.Print("ensuring resource group role assignment")
 
-	res, err := ah.applications.GetServicePrincipalsIDByAppID(ctx, ah.env.FPClientID())
+	fpGraphAuthorizer, err := ah.env.FPAuthorizer(ah.env.TenantID(), ah.env.Environment().GraphEndpoint)
```

Comment:
unhandled error?

Reply:
Removed this part of the code as it was not required. The code is used only in dev environment. No rotation needed.